src/applications/disability-benefits/526EZ/components/ReviewCardField.jsx

```diff
@@ -119,15 +134,19 @@ export default class ReviewCardField extends React.Component {
   getReviewView = () => {
     const ViewComponent = this.props.uiSchema['ui:options'].viewComponent;
     const title = this.getTitle();
+    const description = this.getDescription();
 
     return (
-      <div className="review-card">
-        <div className="review-card--header">
-          <h4 className="review-card--title">{title}</h4>
-          <button className="usa-button-secondary edit-button" onClick={this.startEditing}>Edit</button>
-        </div>
-        <div className="review-card--body">
-          <ViewComponent formData={this.props.formData}/>
+      <div>
+        {description}
```

Comment:
Could we include this logic in the render function instead so that we can reduce duplication? I think something like:
```
   render() {
     let content = this.getReviewView();
     if (this.state.editing) {
       content = this.getEditView();
     }
 
     return (
      <div>
        <description/>
        <content/>
       </div>
     );
```

Reply:
Yeah, that stood out to me. I decided to keep it the way it is because I'd have to make more changes to the `render()` function that would impact code that isn't directly related to this change. That being said, why not :)

Might as well clean things up a bit.